qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -80,7 +80,8 @@ def signature(self) -> 'Signature':
                     for reg in self.gate.signature
                 ]
             )
-        return Signature([Register('qubits', shape=cirq.num_qubits(self.gate), bitsize=1)])
+        shape = cirq.num_qubits(self.gate)
+        return Signature([Register('qubits', shape=shape, bitsize=1)]) if shape else Signature([])
```

Comment:
should `shape` be called `n_qubits`? is it always an integer? If it is: the condition should be `if shape > 0`

Reply:
Yes, it's always an integer. Changed.